src/incubator/TextField/index.tsx

```diff
@@ -187,12 +187,13 @@ const TextField = (props: InternalTextFieldProps) => {
                 />
               )}
               <Input
+                placeholderTextColor={floatingPlaceholder ? 'transparent' : undefined}
                 {...others}
                 style={[typographyStyle, colorStyle, others.style]}
                 onFocus={onFocus}
                 onBlur={onBlur}
                 onChangeText={onChangeText}
-                placeholder={floatingPlaceholder ? undefined : placeholder}
+                placeholder={placeholder}
```

Comment:
Was that required eventually?  
I see the flexG enough fix the issue..

Reply:
It doesn't fix it.. we have to pass the placeholder to set the actual width